### ISSUE TITLE
The test files are missing

### ISSUE THREAD

**Issue by Kawinnat** [Ensembl/plant-scripts#7]
Hi,

I could not find the folder  "/files/test_rice" , the test dataset for the script get_pangene.py.

Thanks,
Pimmy

**Comment by brunocontrerasmoreira** [comment]
Hi @Kawinnat ,
the install instructions for GET_PANGENES are at https://github.com/Ensembl/plant-scripts#pangenes
There are two required steps only after moving to the cloned repo:

    cd plant-scripts
    make install
    make install_pangenes

The second make call will download and unpack a test rice dataset which will be located at *files/test_rice/*
You might want to inspect the [Makefile](https://github.com/Ensembl/plant-scripts/blob/master/Makefile) for more details. 
After those steps are completed you can test whether everything is in place with:

    perl pangenes/get_pangenes.pl -v 
    make test_pangenes

In summary, you get *files/test_rice/* by running:

    make install_pangenes 

and you can remove it with: 

    make uninstall_pangenes
